contracts/transfer/circuits/src/send_to_contract_obfuscated.rs

```diff
@@ -0,0 +1,7 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+//
+// Copyright (c) DUSK NETWORK. All rights reserved.
+
+// TODO The message structure is not yet specified
```

Comment:
If the plan is to merge without implementing this, I would suggest to open an issue pointing to this line.

Reply:
^

Reply:
Agreed, issue #155 opened